BedrockServer.cpp

```diff
@@ -995,7 +999,7 @@ void BedrockServer::_resetServer() {
 }
 
 BedrockServer::BedrockServer(const SData& args)
-  : SQLiteServer(""), _args(args), _requestCount(0), _replicationState(SQLiteNode::SEARCHING),
+  : SQLiteServer(""), shutdownWhileDetached(false), _args(args), _requestCount(0), _replicationState(SQLiteNode::SEARCHING),
```

Comment:
nab, but this is the only arg that isn't `_` prefixed, even though others are also exposed to be updated outside of the BedrockServer object.

Reply:
This is the only public one (the rest are only exposed through getter/setters). I'm fine with it this way.